backend/api/Startup.cs

```diff
@@ -236,15 +237,15 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env, IApiVers
 
             app.UseSwagger(options =>
             {
-                options.RouteTemplate = "/swagger/{documentname}/swagger.json";
+                options.RouteTemplate = "/api-docs/swagger/{documentname}/swagger.json";
             });
             app.UseSwaggerUI(options =>
             {
                 foreach (var description in provider.ApiVersionDescriptions)
                 {
-                    options.SwaggerEndpoint($"/swagger/{description.GroupName}/swagger.json", description.GroupName);
+                    options.SwaggerEndpoint($"/api-docs/swagger/{description.GroupName}/swagger.json", description.GroupName);
                 }
-                options.RoutePrefix = "swagger";
+                options.RoutePrefix = "api-docs";
```

Comment:
You should try changing the route prefix to `api` - we don't have "api-docs" in our reverse proxy in DEV

As per this [article](http://eatcodelive.com/2017/05/19/change-default-swagger-route-in-an-asp-net-core-web-api/), the changes should be:

**app.UseSwagger() changes:**

```csharp
 app.UseSwagger(c =>
 {
      c.RouteTemplate = "api/swagger/{documentName}/swagger.json";
 });
```
**app.UseSwaggerUI() changes:**

```csharp
 app.UseSwaggerUI(c =>
 {
      c.SwaggerEndpoint($"/api/swagger/{description.GroupName}/swagger.json", description.GroupName);
      c.RoutePrefix = "api/swagger";
 });
```